tests/phpunit/tests/user.php

```diff
@@ -2157,6 +2157,44 @@ public function test_send_confirmation_on_profile_email_html_entities_decoded()
 		$this->assertStringNotContainsString( '&#039;Test&#039; blog&#039;s &quot;name&quot; has &lt;html entities&gt; &amp;', $email->subject, 'Email subject does contains HTML entities' );
 	}
 
+	/**
+	 * Check array is a sequential.
+	 *
+	 * @param array $arr array.
```

Comment:
This could do with an improved description for what is been checked and the parameter value.